content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/use_locations.rst

```diff
@@ -2,60 +2,114 @@
 Locations
 =========
 
-A *location* is a specific space within a warehouse. This can be a shelf, room, aisle, etc. There
-are three types of locations in Odoo:
-
-- *Physical locations* are spaces within a warehouse owned by the user's company. These can be a an
-  area where items are stored like an aisle or shelf, or an area where operations take place, like
-  loading and unloading bays.
-- *Partner locations* are the same as physical locations except that they exist within the warehouse
-  of a customer or vendor.
-- *Virtual locations* are locations that do not exist physically, but where items that are not in
-  inventory can be placed. These can be items that have not yet entered inventory, like products
-  that are on the way to a warehouse, or items that are no longer in inventory due to loss or other
-  factors.
-
-.. important::
-   In order to use locations, the :guilabel:`Storage Locations` setting must be enabled. To do so,
-   navigate to :menuselection:`Inventory --> Configuration --> Settings`, scroll down to the
-   :guilabel:`Warehouse` heading, and enable the :guilabel:`Storage Locations` checkbox.
+In Odoo *Inventory*, a *location* is a specific space within a warehouse. This can be a shelf, room,
+aisle, rack, bin, etc.
```

Comment:
There are also virtual and external locations, too! If you'd like to focus on the internal locations, I'd rewrite this to something like:
```suggestion
In Odoo *Inventory*, a *location* is a specific space within the supply chain. Locations within the warehouse (e.g. shelf, room,
aisle, rack, bin, etc) can be used to .... [your next sentence]
```

Reply:
Added a sentence about virtual and external locations. Suggestion implemented ✅